### PR TITLE
for Mono use Miguel's components.md text

### PR DESCRIPTION
**Pull request by tdykstra**

### CODE REVIEW

File: docs/standard/glossary.md [hunk 1]
```diff
@@ -135,13 +135,15 @@ See [Packages, Metapackages and Frameworks](../core/packages.md)
 
 ## Mono
 
-An open source alternative to the .NET Framework.
+Mono is a .NET implementation that is mainly used when a small runtime is required. It is the runtime that powers Xamarin applications on Android, Mac, iOS, tvOS and watchOS and is focused primarily on a small footprint.
```
Comment (rpetrusha) [inline docs/standard/glossary.md]: nit: on a small footprint --> apps that require a small footprint

**Review by rpetrusha — APPROVED** [review]
LGTM, except for one nit.

**Review by mairaw — APPROVED** [review]
LGTM

**Comment by mairaw** [comment]
Build seemed stuck, Closed and reopened to try again. @tdykstra, do we need @terrajobst review before merging?

**Comment by tdykstra** [comment]
@mairaw No - already reviewed for the components doc.

**Pull request merged**